tests/testthat/_snaps/verb-expand.md

```diff
@@ -109,14 +115,23 @@
     Output
       <SQL>
       SELECT `x`, `y`, COALESCE(`z`, 'c') AS `z`
-      FROM (SELECT COALESCE(`LHS`.`x`, `RHS`.`x`) AS `x`, COALESCE(`LHS`.`y`, `RHS`.`y`) AS `y`, `z`
-      FROM (SELECT `x`, `y`
-      FROM (SELECT DISTINCT `x`
-      FROM `df`) `LHS`
-      CROSS JOIN (SELECT DISTINCT `y`
-      FROM `df`) `RHS`
-      ) `LHS`
-      FULL JOIN `df` AS `RHS`
-      ON (`LHS`.`x` = `RHS`.`x` AND `LHS`.`y` = `RHS`.`y`)
+      FROM (
+        SELECT
+          COALESCE(`LHS`.`x`, `RHS`.`x`) AS `x`,
+          COALESCE(`LHS`.`y`, `RHS`.`y`) AS `y`,
+          `z`
+        FROM (
+          SELECT `x`, `y`
+          FROM (
+            SELECT DISTINCT `x`
+            FROM `df`
+          ) `LHS`
+          CROSS JOIN (
+            SELECT DISTINCT `y`
+            FROM `df`
+          ) `RHS`
+        ) `LHS`
+        FULL JOIN `df` AS `RHS`
+        ON (`LHS`.`x` = `RHS`.`x` AND `LHS`.`y` = `RHS`.`y`)
```

Comment:
Why don't the parens change here?

Reply:
`sql_query_join.DBIConnection()` still uses `sql_clause("ON", on, sep = " AND", parens = TRUE)` which doesn't add any parentheses.
But `sql_query_semi_join.DBIConnection()` uses `sql_clause_where(on, lvl = 1)` which adds the parentheses.
Would you prefer to also have parens here?

Reply:
No strong preference, but it feels better to be consistent?